### PR TITLE
Simplify code by removing handler list

### PR DESCRIPTION
**Pull request by DeltaEvo**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -61,17 +49,16 @@ function handle_login({ world, handlers }) {
...
 
     login(state)
     send_resource_pack(state)
     position_change_event(state)
     chunk_change_event(state)
     update_chunks(state)
-    for (const handler of handlers) handler(state)
+    spawn_villager(state)
+    open_trade(state)
+    spawn_mob(state)
```
Comment (Sceat) [inline src/index.js:61]: you could here use the reducer to avoid repeating `state` but feels ok anyway tho

File: src/trade/trade.js [hunk 2]
```diff
@@ -2,60 +2,63 @@ import { version } from '../settings.js'
...
-              inputItem2: {
-                present: true,
-                itemId: diamond,
-                itemCount: 1,
-              },
-              maximumNbTradeUses: 99999,
+export function open_trade({ client, world }) {
+  const { villagers, windowId } = world.trade
+  client.on('use_entity', ({ target, mouse, sneaking }) => {
+    if (villagers.includes(target) && mouse === 2 && sneaking === false) {
```
Comment (Sceat) [inline src/trade/trade.js:28]: Magic `2` ? 😏

**Review by Sceat — CHANGES_REQUESTED** [review]

File: src/trade/trade.js (hunk 2 above)
Comment (DeltaEvo) [reply src/trade/trade.js:28]: It's from the original code cc @Mathis-Paul-62 I will create an issue for that

**Review by Sceat — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (DeltaEvo) [reply src/index.js:61]: For now since we not have much handlers I think I will keep this, we will refactor this when we have more functions

**Pull request merged**